README.md

```diff
@@ -1,32 +1,81 @@
 Mattermost Kubernetes 
 ==========================
 
-# This is an experimental work in progress
+# Getting started using minikube
 
-#### XXX TODO FIX ME - Need to document setup process
+## Install minikube and kubectl
 
-This projects uses Kubernetes and Helm
+See: https://kubernetes.io/docs/tasks/tools/install-minikube/
 
-1. Modify values.yaml
-2. run `make` to build the helm package
-3. run `make install` to install the helm package
+## Launch minikube
 
-Helm repository for latest master of this repository is available, to add:
+The helm charts start a lot of containers, and it will work better if you 
+launch minikube with additional memory and CPU. You also need to enable 
+persistent volume mapping. This only needs to be done the first time you launch
+minikube. The settings will persist across restarts. If you need to modify the
+values try `minikube delete` and `minikube stop`
 
-`helm repo add mattermost https://releases.mattermost.com/helm`
+```bash
+minikube start --memory 4096 --cpus 4 --mount
+```
+
+## Install and start Helm
+
+See: https://docs.helm.sh/using_helm/#quickstart
+
+Once helm is installed, run `helm init` to get it loaded onto minikube
+
+You may need to add some additional repos for helm
+
+```bash
+helm repo add mattermost https://releases.mattermost.com/helm
+helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+```
+
+## Launch Mattermost
+
+The helm charts have external dependencies, fetch them with:
+
+```bash
+helm dependency update
```

Comment:
Is this common practice or do people usually check in their dependencies?
Also this isn't necessary yet because we don't actually have any dependencies that are being fetched.